StrawberryShake.Server.targets cannot evaluate ::VersionGreaterThanOrEquals(' ', 6)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Product

Strawberry Shake

### Describe the bug

When following this documentation (https://chillicream.com/docs/strawberryshake/v13/get-started/console), I get the following error when trying to build (sub-step 5 of step 4):
`StrawberryShake.Server.targets(38, 20): [MSB4184] The expression "[MSBuild]::VersionGreaterThanOrEquals('', 6)" cannot be evaluated. Version string was not in a correct format.`.

I have followed the exact same instructions, so I don't understand why I get the error message. This is the generated code where it gives an error:

`    <PropertyGroup>
      <DotNetMajor Condition="$([MSBuild]::VersionGreaterThanOrEquals($(DotNetVersion),6))">6</DotNetMajor>
      <DotNetMajor Condition="$([MSBuild]::VersionGreaterThanOrEquals($(DotNetVersion),7))">7</DotNetMajor>
    </PropertyGroup>
`

The error mentions this file name: _StrawberryShake.Server.Targets_

### Steps to reproduce

1. Follow the instructions fromhttps://chillicream.com/docs/strawberryshake/v13/get-started/console
2. Build the application when the instruction says _5. Compile your project._

### Relevant log output

```shell
`StrawberryShake.Server.targets(38, 20): [MSB4184] The expression "[MSBuild]::VersionGreaterThanOrEquals('', 6)" cannot be evaluated. Version string was not in a correct format.`.
```


### Additional Context?

_No response_

### Version

13.1.0

I have this as well.

Is this because the `<Exec>` blocks are not evaluated, but the `DotNetMajor` properties are evaluated? That would be weird.
This is what is logged:

```
   1:7>Project "C:\Projects\XXXXXX.csproj" on node 1 (default targets).
     1>_GenerateGraphQLCode:
       Skipping target "_GenerateGraphQLCode" because all output files are up-to-date with respect to the input files.
     1>C:\Users\XXXXX\.nuget\packages\strawberryshake.server\13.1.0\build\StrawberryShake.Server.targets(39,20): error MSB4184: The expression "[MSBuild]::VersionG
       reaterThanOrEquals('', 6)" cannot be evaluated. Version string was not in a correct format. [C:\Projects\XXXXX.csproj]
```

I also get:

```
1>MSBUILD : warning MSB5029: The value "\**\*.cs" of the "Include" attribute in element <ItemGroup> in file "C:\Users\XXXXX\.nuget\packages\strawberryshake.server\13.1.0\build\StrawberryShake.Server.targets (23,16)" is a wildcard that results in enumerating all files on the drive, which was likely not intended. Check that referenced properties are always defined.
```

Apparently more variables are empty.

--- a/src/StrawberryShake/MetaPackages/Common/MSBuild/StrawberryShake.targets
+++ b/src/StrawberryShake/MetaPackages/Common/MSBuild/StrawberryShake.targets
@@ -24,15 +24,18 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="_ParseVersion">
+    <Exec Command="dotnet --version" StandardOutputImportance="Low" ConsoleToMsBuild="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="DotNetVersion" />
+    </Exec>
+  </Target>
+
   <Target
     Name="_GenerateGraphQLCode"
     Inputs="@(GenerateGraphQLCodeItems)"
     Outputs="$(MSBuildProjectDirectory)\$(IntermediateOutputPath)berry\.build.info"
+    DependsOnTargets="_ParseVersion"
     Condition="'@(GraphQL)' != ''">
-
-    <Exec Command="dotnet --version" StandardOutputImportance="Low" ConsoleToMsBuild="true">
-      <Output TaskParameter="ConsoleOutput" PropertyName="DotNetVersion" />
-    </Exec>
 
     <PropertyGroup>
       <DotNetMajor Condition="$([MSBuild]::VersionGreaterThanOrEquals($(DotNetVersion), 6))">6</DotNetMajor>

